test/cli.test.ts

```diff
@@ -44,6 +43,38 @@ describe('CLI Tests', () => {
       throw new Error(stderr);
     }
   });
+  it(`Can specify dependency list`, async () => {
+    const GRAMMAR_FILE = path.join(ROOT_DIR, 'examples/minimal-with-dep.pegjs');
+    const outTsName = path.join(ROOT_DIR, 'output/minimal-with-dep.ts');
+    const barSource = path.join(ROOT_DIR, 'examples/bar.ts');
+    const barDest = path.join(ROOT_DIR, 'output/bar.ts');
+    // Copy the dependency to the output directory
+    await exec(`cp "${barSource}" "${barDest}"`);
+    {
+      // Create the parser
+      const { stdout, stderr } = await exec(
+        `npx peggy --plugin "${PLUGIN_PATH}" --dependency foo:./bar -o "${outTsName}" "${GRAMMAR_FILE}"`
```

Comment:
It's worth trying with `--dependencies "{\"foo\": \"./bar\"}"` or a config file with a `dependencies` key, as well.

Reply:
I think this is okay, since Peggy merges `--dependency` and `--dependencies` together for us, we'll trust Peggy to do it right.